ml-agents/mlagents/envs/environment.py

```diff
@@ -244,7 +245,7 @@ def reset(self, config=None, train_mode=True) -> AllBrainInfo:
 
         if self._loaded:
             outputs = self.communicator.exchange(
-                self._generate_reset_input(train_mode, config)
+                self._generate_reset_input(self.train_mode, config)
```

Comment:
Can we just change the signature of this method to use train_mode which is a member of the UnityEnvironment class? i.e.`self._generate_reset_input(config)`?